contracts/mocks/HoldableMock.sol

```diff
@@ -15,6 +17,18 @@ contract HoldableMock is Holdable {
         holds[operationId.toHash()].expiration = _expiration;
     }
 
+    function executeHoldToBurn(string calldata operationId) external {
```

Comment:
Why does this new method appear?

Reply:
This method is added to test the new parameter introduced at `_executeHold` called `doTransfer`.

This method is internal, and the only way to test it is to introduce a new public or external method who use it and create a test with it.